configuring/configuring-log-forwarding.adoc

```diff
@@ -113,8 +113,31 @@ Filters are configured in an array under `spec.filters`. They can match incoming
 
 Administrators can configure the following types of filters:
```

Comment:
This line is now left on its own in the preview - is this intentional? maybe enumerate the filter types here?

> Filters
> Filters are configured in an array under spec.filters. They can match incoming log messages based on the value of structured fields and modify or drop them.
> 
> Administrators can configure the following types of filters:
> 
> Enabling multi-line exception detection